auslib/db.py

```diff
@@ -1207,8 +1207,13 @@ def enactChange(self, sc_id, enacted_by, transaction=None):
         # when the transaction is rolled back.
         # We explicitly avoid using ScheduledChangeTable's update() method here
         # because we don't want to trigger its validation of conditions. Doing so
-        # would raise any exception for any timestamp based changes, because
-        # they are already in the past when we're ready to enact them.
+        # would raise any exception for any timestamp based changes, be
+        # Updating in conditions table also so that heistory view can work
```

Comment:
What happened with this comment? Line 1210 has an abruptly ended sentence now. Please fix the typo, too ("heistory" vs "history").